content/en/docs/usecases/validation.md

```diff
@@ -47,7 +47,7 @@ The `cue vet` command can verify that the values in `ranges.yaml`
 are correct by just mentioning the two files on the command line.
 
 {{< highlight none >}}
-$ cue vet checks.cue
+$ cue vet ranges.yaml checks.cue
 max: invalid value 5 (out of bound >10):
     ./check.cue:2:15
     ./data.yaml:5:7
```

Comment:
The output in this line seem's to be incorrect as well.

Reply:
not sure what happened there, but should be the same name as well indeed.

Reply:
yes, as you said it should be `ranges.yaml`, not `data.yaml`.
Additionally, in my local, results is displayed as `./check.cue:2:16` instead of `./check.cue:2:16`. So I modified it that way.